src/main/java/uk/gov/hmcts/reform/sandl/snlapi/security/JwtAuthenticationFilter.java

```diff
@@ -18,7 +18,7 @@
 
 public class JwtAuthenticationFilter extends OncePerRequestFilter {
 
-    private static final Logger logger = LoggerFactory.getLogger(JwtAuthenticationFilter.class);
+    private static final Logger localLogger = LoggerFactory.getLogger(JwtAuthenticationFilter.class);
```

Comment:
what is the common convention used in other classes across all projects here?

Reply:
Can't find any, the thing here is that SonarQube noticed we are extending a class which already have a field named `logger` and it's of a different type and it's a code smell.
As we want to log with proper class info I prefer to change field name here.